inst/htmljs/animint.js

```diff
@@ -125,24 +125,71 @@ var animint = function (to_select, json_file) {
   var measureText = function(pText, pFontSize, pAngle, pStyle) {
     if (pText === undefined || pText === null || pText.length === 0) return {height: 0, width: 0};
     if (pAngle === null || isNaN(pAngle)) pAngle = 0;
-
+    
+    // Create temporary container to measure text
     var container = element.append('svg');
-    // do we need to set the class so that styling is applied?
-    //.attr('class', classname);
-
-    container.append('text')
-      .attr({x: -1000, y: -1000})
+    var textElement = container.append('text')
       .attr("transform", "rotate(" + pAngle + ")")
       .attr("style", pStyle)
-      .attr("font-size", pFontSize)
-      .text(pText);
-
+      .attr("font-size", pFontSize);
+    
+    // Check if text contains <br/> tags (multi-line)
+    var textStr = String(pText || '');
+    var lines = textStr.split('<br/>');
+    
+    if (lines.length > 1) {
+      // Multi-line: use setMultilineText helper to render properly
+      setMultilineText(textElement, pText);
+    } else {
+      // Single line: simple text rendering
+      textElement.text(pText);
```

Comment:
it seems unusual that there needs to be a special case for multiple lines.

also this if length > 1 else pattern is repeated in the setMultilineText function.

please simplify